### PR TITLE
Add missing NEWs fragments

### PR DESCRIPTION
**Pull request by jieru-hu**
Add news fragments for some of the closed issues.

### CODE REVIEW

**Comment by jieru-hu** [comment]
two files are empty, will fix them.

File: news/1449.docs [hunk 1]
```diff
@@ -0,0 +1 @@
+Documentation for `hydra.searchpath`
```
Comment (omry) [inline news/1449.docs:1]: Duplicate.

File: news/1377.api_change [hunk 2]
```diff
@@ -0,0 +1 @@
+Support partial failures in MULTIRUN mode
```
Comment (omry) [inline news/1377.api_change:1]: Add a newline at the end of news fragments.

File: news/1461.bugfix [hunk 3]
```diff
@@ -0,0 +1 @@
+`--cfg=hydra` now works for top-level read-only config
```
Comment (omry) [inline news/1461.bugfix:1]: The key point is read-only.

```
--cfg=hydra now works when config is read-only
```

File: news/1529.feature [hunk 4]
```diff
@@ -0,0 +1 @@
+Add `env` to Hydra's config group
```
Comment (omry) [inline news/1529.feature:1]: Let's keep this one internal. Without documentation this means absolutely nothing to users and I don't think we should document it right now.

**Review by omry — COMMENTED** [review]
In general I have been pretty sparse with news fragments about documentation changes. I try to keep them limited to large items only, not minor changes.

File: news/1582.docs [hunk 5]
```diff
@@ -1 +1 @@
-Documentation for configuring plugin
\ No newline at end of file
+Documentation for configuring plugin
```
Comment (Jasha10) [inline news/1582.docs:1]:
```suggestion
Document how to configure Hydra plugins
```

**Review by Jasha10 — COMMENTED** [review]
Just one small suggestion:

**Pull request merged**